### DISCUSSION TITLE
Ikea Tradfi bulbs power_on_behavior brightness

### DISCUSSION THREAD

**Discussion by thomasfjen** [Koenkk/zigbee2mqtt#8814] [Devices]
Is it possible to set a custom brightness setting similiar to the power_on_behavior? 
I use the bulbs as the main light in two rooms connected to a "dumb" wall switch which cuts power to them and just sometimes bring the brightness down or turn them off via remote. 
When reentering the room i would like to have the brightness set to 100% everytime similiar to the power_on_behavior.

At first the bulb was connected directly to the IKEA E1743 dimmer without any instance of Z2M and it was the standard behavior, so there could be a command to it am i right? 

Thanks in Advance!

**Comment by mfalkvidd** [comment]
@thomasfjen did you find a solution or workaround for this? I have a similar use case.

**Reply by thomasfjen** [reply]
Sadly not, but haven't looked into that for a long time.
